Makefile

```
@@ -0,0 +1,351 @@
+# Makefile for osg-test
+
+
+# ------------------------------------------------------------------------------
+# Release information: Update for each release
+# ------------------------------------------------------------------------------
+
+PACKAGE := htcondor-ce
+VERSION := 5.0.0
+
+
+# ------------------------------------------------------------------------------
+# Other configuration: May need to change for a release
+# ------------------------------------------------------------------------------
+
+PYTHON			:= /usr/bin/python3
+
+INSTALL_BIN_DIR		:= usr/bin
+INSTALL_LIB_DIR         := usr/lib
+INSTALL_SHARE_DIR	:= usr/share
+INSTALL_STATE_DIR	:= var
+INSTALL_SYSCONF_DIR	:= etc
+INSTALL_WSGI_DIR        := $(INSTALL_STATE_DIR)/www/wsgi-scripts
+INSTALL_PYTHON_DIR	:= $(shell $(PYTHON) -c 'from distutils.sysconfig import get_python_lib; print(get_python_lib())')
```

Apparently distutils is going away in Python 3.12. Use (works in both 3 and 2.7):
```suggestion
INSTALL_PYTHON_DIR	:= $(shell $(PYTHON) -c 'from sysconfig import get_paths; print(get_paths()["purelib"])')
```